docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -115,26 +115,36 @@ The exported file is automatically named using the format `export_YYYYMMDDHHMMSS
 | `-k`, `--key`            | string  | Passes the encryption key as part of the `export` command. <br/> The `--key` option can't be combined with `--no-encrypt`. |                                |
 | `-f`, `--file`       | string  | Specifies the export filename. Do not include a file extension.                                           |
+| `--exclude` | string | Exclude data using comma-separated data types. `--exclude` types override `--only` types. The available types are: `content`, `files`, and `config`.|
+| `--only`| string | Include only these data. The available types are: `content`, `files`, and `config`.|
 | `-h`, `--help`       |     -    | Displays help for the `strapi export` command.                                                            |
 
 **Examples**
 
 ```bash
 # examples of strapi export:
 
-strapi export -f myData # exports your data with the default options and the filename myData (which will result in a file named myData.tar.gz.enc)
-strapi export --no-encrypt # exports your data without encryption. 
+# export your data with the default options and the filename myData (which will result in a file named myData.tar.gz.enc):
+strapi export -f myData 
+# exports your data without encryption:
+strapi export --no-encrypt
+# export your data and configuration but not media files:
+strapi export --exclude files 
 ```
 
-## strapi import <BetaBadge />
+## strapi import
 
 [Imports data](/developer-docs/latest/developer-resources/data-management.md) into your project. The imported data must originate from another Strapi application. You must pass the `--file` option to specify the filename and location for the import action.
 
-| Option             | Type   | Description                                                               |
-|--------------------|--------|---------------------------------------------------------------------------|
+| Option             | Type   | Description                                                                   |
+|--------------------|--------|-------------------------------------------------------------------------------|
 | `-k,` `--key`          | string | Provide the encryption key in the command instead of a subsequent prompt. |
 | `-f`, `--file`         | string | Path and filename with extension for the data to be imported.             |
-| `-h`, `--help`         |   -     | Display the `strapi import` help commands.                                |
+| `--force`| - |  Automatically answers "yes" to all prompts, including potentially destructive requests      |
+| `--exclude` | string | Exclude data using comma-separated data types. `--exclude` types override `--only` types. The available types are: `content`, `files`, and `config`.|
+| `--only`| string | Include only these data. The available types are: `content`, `files`, and `config`.      |
+
+| `-h`, `--help`         |   -     | Display the `strapi import` help commands.                               |
```

Comment:
There seem to be some issues with spacing here in --force, --exclude, and --only. And I think the whole table doesn't line up with the header columns

Reply:
I reworked the whole table to try to improve the formatting. I will do the same for the export table.